lorem.html

```diff
@@ -43,7 +52,7 @@ <h1>Lorem Ipsum</h1>
                 <p>
                     <img 
                     style="float: right"
-                    src="https://loremflickr.com/320/240/Team, Rocket" 
+                    src="https://loremflickr.com/320/240/Team,+Rocket" 
```

Comment:
Validators don't like seeing spaces in URLs.  Replace spaces with either `+` or `%20`